config/proto_conversions.go

```diff
@@ -85,6 +85,8 @@ func FromProto(proto *pb.RobotConfig, logger logging.Logger) (*Config, error) {
 		cfg.Debug = *proto.Debug
 	}
 
+	logAnyFragmentOverwriteErrors(logger, proto.OverwriteFragmentStatus)
```

Comment:
How does `proto.OverwriteFragmentStatus` get filled in?

Reply:
Thanks for looking at this so quickly!
It gets filled in [here](https://github.com/viamrobotics/app/blob/93c939fc8f142620812e4f54e61effb56c90f58b/robotservice/config_retriever.go#L299).

Specifically, when there is an error in overwriting a fragment whether it's a user error or an internal error.

Reply:
Cool; thank you! Agree with @cheukt then about the colon but otherwise LGTM :)